### PR TITLE
Fix installation check: K8S default scheduler only supports <= 1 type of computing device.

### PR DESCRIPTION
**Pull request by hzy46**

### CODE REVIEW

File: contrib/kubespray/script/validate_layout_schema.py [hunk 1]
```diff
@@ -79,6 +85,11 @@ def check_layout(layout):
     if 'pai-worker' in masters[0] and masters[0]['pai-worker'] == 'true':
         logger.error("One machine can not be pai-master and pai-worker at the same time.")
         return False
+    # if cluster_config.enable_hived_scheduler is false, there should be <= 1 type of computing device
+    if 'enable_hived_scheduler' in cluster_config and cluster_config['enable_hived_scheduler'] is False:
```
Comment (suiguoxin) [inline contrib/kubespray/script/validate_layout_schema.py:89]: Will hived scheduler be enabled if `enable_hived_scheduler` is not set in `config.yaml` ?

File: contrib/kubespray/script/validate_layout_schema.py [hunk 2]
```diff
@@ -79,6 +85,11 @@ def check_layout(layout):
...
+        if len(worker_computing_devices) > 1:
+            logger.error('K8S default scheduler only supports <= 1 type of computing device.')
```
Comment (suiguoxin) [inline contrib/kubespray/script/validate_layout_schema.py:91]: `logger.error('... as workers nodes')` ?
We should recommend the users to use hived in this case

File: contrib/kubespray/script/validate_layout_schema.py (hunk 1 above)
Comment (hzy46) [reply contrib/kubespray/script/validate_layout_schema.py:89]: yes

File: contrib/kubespray/script/validate_layout_schema.py (hunk 2 above)
Comment (hzy46) [reply contrib/kubespray/script/validate_layout_schema.py]: done

**Review by suiguoxin — APPROVED** [review]

**Pull request merged**